src/components/PeoplePage.tsx

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable no-param-reassign */
+import { useEffect, useState } from 'react';
+import { People } from '../Types';
+import { PeopleTable } from './PeopleTable';
+
+export const PeoplePage : React.FC = () => {
+  const url
+  = 'https://mate-academy.github.io/react_people-table/api/people.json';
+  const [people, setPeople] = useState<People[]>();
+
+  useEffect(() => {
+    async function getPeople() {
+      fetch(url)
```

Comment:
Will be better to create separate api.ts file with function for fetching people from server